src/main/python/inverse_parser/_attribute_setter.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Aniskov N.
+import ast
+import xml.etree.ElementTree as ET
+from pydoc import locate
+
+import src.main.python.inverse_parser._node_restorer as nr
+from src.main.python.inverse_parser._xml_node_children_getter import _XmlNodeChildrenGetter
+from src.main.util.const import DEFAULT_ENCODING
+
+
+class _AttributeSetter:
+
+    @staticmethod
+    def set_const_value(xml_node: ET.Element, py_node: ast.AST, py_node_attrib_name: str) -> None:
+        """
+        :param xml_node:
+        :param py_node:
+        :param py_node_attrib_name:
+        """
+        if 'value_type' not in xml_node.attrib:
+            raise RuntimeError(f'missing value_type attribute in {xml_node.tag} node')
+
+        str_val_repr = xml_node.attrib['value']
+        str_val_type_repr = xml_node.attrib['value_type']
+
+        value_type = locate(str_val_type_repr)
+
+        if str_val_type_repr == 'ellipsis':
+            value = ...
+        elif str_val_type_repr == 'NoneType':
+            value = None
+        elif not value_type:
+            raise RuntimeError(f'failed to locate Constant.value type: {xml_node.attrib["value_type"]}')
+
+        elif value_type == bytes:
+            value = value_type(str_val_repr[2:len(str_val_repr) - 1].encode(DEFAULT_ENCODING))
+
+        elif value_type == bool:
+            value = str_val_repr == 'True'
+
+        else:
+            value = value_type(str_val_repr)
```

Comment:
Maybe it would be better to create a map for different values... Then we can change it to something like this:
`value = value_types.get(str_val_type_repr, value_type(str_val_repr))`
I think it looks more clear and we can see key-value pairs together in the dictionary

Reply:
Yes, I agree that this is not the easiest piece of code, but I don't see an obvious way to organize it into one map, as you proposed, since in different cases i infer `value` differently. 
And I don't quite understand how would you like this line to work `value = value_types.get(str_val_type_repr, value_type(str_val_repr))`